src/Resources/translations/flashes.en.yaml

```
@@ -0,0 +1,5 @@
+setono_sylius_order_edit:
+    error:
+        order_update: 'Order should not have bigger total than before editing'
+    success:
+        order_update: 'Order has been successfully updated'
```

Is this a normal structure? I would have expected something like this:

```suggestion
setono_sylius_order_edit:
    order_update:
        error: 'Order should not have bigger total than before editing'
        success: 'Order has been successfully updated'
```

---

I probably had a sudden attack of stupidity when writing this XD totally agreed 🖖